Jenkinsfile

```diff
@@ -534,6 +534,24 @@ pipeline {
                     }
                 }
 
+                stage('Hip Implicitgemm layouts Release') {
+                    agent{ label rocmnode("gfx908") }
```

Comment:
Tests should use Vega agent (which means Vega 10 or Vega 20). Fixed.

Reply:
Thanks! Just want to make sure I understand it right: When there are MLIR based xdlops kernels as well as those regular vega kernel, should I flip the `rocmnode` back to `gfx908` (because gfx908 is capable of running both flavors)?

Reply:
You will add another stage that will run on gfx908.